### PR TITLE
MGDCTRS-592: feat: Prepare the UI for namespace related backend changes

### PR DESCRIPTION
**Pull request by indraraj**
Closes issue https://issues.redhat.com/browse/MGDCTRS-592

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#stage
 BASE_PATH=https://wxn4aqqc8bqvxcy6unfe.api.stage.openshift.com
-#BASE_PATH=https://cos-fm.apps.kammellol.rhmw-integrations.net
+#dev
+#BASE_PATH=https://cos-fleet-manager-cos.rh-fuse-153f1de160110098c1928a6c05e19444-0000.eu-de.containers.appdomain.cloud
```
Comment (indraraj) [inline .env:4]: @gashcrumb you may have to test it against the dev cluster here

Comment (gashcrumb) [reply .env:4]: :+1: yep, thanks!

File: src/apis/api.ts [hunk 2]
```diff
@@ -52,6 +52,10 @@ export type FetchCallbacks<RawDataType> = (
   onError: (errorMsg: string) => void
 ) => () => void;
 
+// interface ConnectorWithNameSpace extends Connector {
+//   namespace_id: string;
+// }
```
Comment (gashcrumb) [inline src/apis/api.ts:57]: Worth deleting this, personally I never trust that commented out code in a repo is ever going to be helpful :smile:

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx [hunk 3]
```diff
@@ -236,16 +236,18 @@ export const CreateConnectorWizard: FunctionComponent<CreateConnectorWizardProps
       },
       kafkaInstanceStep,
       {
-        name: t('OSD cluster'),
-        isActive: state.matches('selectCluster'),
+        name: t('Namespace'),
+        // isActive: state.matches('selectCluster'),
```
Comment (gashcrumb) [inline src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx:240]: comment

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx [hunk 4]
```diff
@@ -294,7 +296,8 @@ export const CreateConnectorWizard: FunctionComponent<CreateConnectorWizardProps
           send('jumpToSelectKafka');
           break;
         case 3:
-          send('jumpToSelectCluster');
+          // send('jumpToSelectCluster');
```
Comment (gashcrumb) [inline src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx:299]: same here

**Comment by gashcrumb** [comment]
@indraraj also, any areas in this PR you'd like extra attention on when reviewing?

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx [hunk 5]
```diff
@@ -236,16 +236,18 @@ export const CreateConnectorWizard: FunctionComponent<CreateConnectorWizardProps
...
       {
-        name: t('OSD cluster'),
-        isActive: state.matches('selectCluster'),
+        name: t('Namespace'),
+        // isActive: state.matches('selectCluster'),
+        isActive: state.matches('selectNamespace'),
         component: (
           <StepErrorBoundary>
-            <SelectCluster />
+            {/* <SelectCluster /> */}
```
[the remark this answers is not in the document]
Comment (indraraj) [reply src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx:244]: @gashcrumb yeah, I may need to clean the PR in general. Thanks

File: src/apis/api.ts (hunk 2 above)
Comment (indraraj) [reply src/apis/api.ts]: done

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx (hunk 3 above)
Comment (indraraj) [reply src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx]: done

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (indraraj) [reply src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx]: done

File: src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx (hunk 4 above)
Comment (indraraj) [reply src/app/components/CreateConnectorWizard/CreateConnectorWizard.tsx]: done

**Comment by indraraj** [comment]
Cleared the leftover commented code

**Review by gashcrumb — APPROVED** [review]
LGTM, and tested locally, works for me.  Thanks!

**Pull request merged**